app/util/AWS.scala

```diff
@@ -87,6 +100,22 @@ class AWSConfig @Inject() (config: Configuration) {
     null
   )
 
+  private def createUploadSTSClient() = {
+    val provider = stage match {
+      case "DEV" =>
+        // Only required in dev. Instance profile credentials are sufficient when deployed
+        val accessKey = config.getString("aws.upload.accessKey").get
```

Comment:
Federated users can't make STS assume role requests unfortunately. I believe this is on the roadmap